services/addons/images/count_metric/gen_email.py

```diff
@@ -0,0 +1,103 @@
+import logging
+import os
+import daily_emailer

```

Comment:
Pretty sure we handled this in another PR with the emailer circular reference?

Reply:
I believe this is now resolved.